include/vigra/config.hxx

```diff
@@ -149,7 +149,7 @@
 //                                                       //
 ///////////////////////////////////////////////////////////
 
```

Comment:
Can you add `#define VIGRA_NO_INT_ISNAN` at the appropriate places (i.e. for the platforms where we already know that it is required)? This way, most people using VIGRA as a dependency won't be forced to run your `include(VigraConfigCmath)` in their cmake files.